projects/packages/videopress/src/client/admin/components/site-settings-section/index.tsx

```diff
@@ -23,9 +24,16 @@ import type React from 'react';
  */
 const SiteSettingsSection: React.FC< SiteSettingsSectionProps > = ( {
 	videoPressVideosPrivateForSite,
+	siteIsPrivate,
+	siteType,
 	onPrivacyChange,
 } ) => {
 	const { canPerformAction } = usePermission();
+	const siteIsAtomicPrivate = siteIsPrivate && siteType === SITE_TYPE_ATOMIC;
+	const disablePrivacyToggle = ! canPerformAction || siteIsAtomicPrivate;
+	const disabledReason = siteIsAtomicPrivate
+		? __( 'You cannot change this setting because your site is private. You can only choose the video privacy default on public sites.', 'jetpack-videopress-pkg' )
```

Comment:
I think you need to fix eslint issues:

```suggestion
		? __(
				'You cannot change this setting because your site is private. You can only choose the video privacy default on public sites.',
				'jetpack-videopress-pkg'
		  )
```